src/Route/RoutePageGenerator.php

```diff
@@ -249,4 +239,39 @@ protected function writeln(OutputInterface $output = null, $message)
             $output->writeln($message);
         }
     }
+
+    /**
+     * Generate root page (parent for another generated pages).
```

Comment:
```suggestion
     * Generate root page (parent for another generated page).
```

Reply:
Usually, there are couple of pages, so I think it's correct.

Reply:
It isn't, because "another" is singular. So it's either "another page" or "other pages"

Reply:
Thanks, understood, one minute.

Reply:
@greg0ire could you check again?